src/linux_syscall_support_unittest.cc

```diff
@@ -140,42 +140,43 @@ static void SigAction(int signum, siginfo_t *si, void *arg) {
 
 static void Sigaction() {
   puts("Sigaction...");
+  const size_t kSigsetSize = sizeof(struct kernel_sigset_t);
   int signum       = SIGPWR;
   for (int info = 0; info < 2; info++) {
     signaled         = 0;
     struct kernel_sigaction sa = ZERO_SIGACT, old, orig;
-    CHECK(!sys_sigaction(signum, NULL, &orig));
+    CHECK(!sys_rt_sigaction(signum, NULL, &orig, kSigsetSize));
```

Comment:
Just curious if we've verified that this form of syscall is supported in the older kernels we support: we still say we'll run with RHEL 7.5 for example as our oldest kernel.

Reply:
The sources of RHEL 7.5 (x86-32 and x86-64) contain the declarations of these sys_rt_ functions.
I hope arm a64 will also contain these declarations.